futures-util/src/try_stream/try_for_each.rs

```diff
@@ -0,0 +1,63 @@
+use core::marker::Unpin;
+use core::mem::PinMut;
+use futures_core::future::{Future, TryFuture};
+use futures_core::stream::TryStream;
+use futures_core::task::{self, Poll};
+
+/// A stream combinator which attempts to execute a unit closure over each
```

Comment:
What's a "unit closure"?

Reply:
Okay I see, but it seems odd. I'm a bit torn whether these short descriptions make sense or not. I removed them from the TryFuture combinator types, but others still have them

Reply:
a closure that returns unit. Changed.

Reply:
async closure

Reply:
Yeah, that's what I changed it to.